### PR TITLE
button: Improve a11y

### PR DESCRIPTION
**Pull request by hcodes**

### CODE REVIEW

File: common.blocks/button/button.js [hunk 1]
```diff
@@ -116,7 +116,7 @@ provide(BEMDOM.decl({ block : this.name, baseBlock : Control }, /** @lends butto
     },
 
     _onKeyDown : function(e) {
-        if(this.hasMod('disabled')) return;
+        if(this.hasMod('disabled') || e.shiftKey || e.ctrlKey || e.altKey || e.metaKey) return;
```
Comment (dfilatov) [inline common.blocks/button/button.js]: How are these changes related with a11y?

**Comment by hcodes** [comment]
https://lego.yandex-team.ru/a11y/keyboard-management/
Модификаторы

**Comment by hcodes** [comment]
Fixed.

File: common.blocks/button/_type/button_type_link.js [hunk 2]
```diff
@@ -23,12 +23,16 @@ provide(Button.decl({ modName : 'type', modVal : 'link' }, /** @lends button.pro
         'disabled' : {
             'true' : function() {
                 this.__base.apply(this, arguments);
+
                 this.domElem.removeAttr('href');
+                this.domElem.attr('aria-disabled', true);
```
Comment (narqo) [inline common.blocks/button/_type/button_type_link.js:28]: ~~Can't this two calls be chained?~~

Comment (narqo) [inline common.blocks/button/_type/button_type_link.js:28]: 1) Why do we do this only for `button_type_link`? What about `button` itself?
2) Can we do it in the `control` base class? Does this behaviour differ from block to block?

**Comment by narqo** [comment]
What about initial disabled state? What if `_disabled` was set in BEMJSON?
